docs/writing-tests/testdriver-extension-tutorial.md

```diff
@@ -15,6 +15,31 @@ First, we need to think of what the API will look like a little. We will be usin
 
 The first part of this will be browser agnostic, but later we will need to implement a specific layer for each browser (here we will do Firefox and Chrome).
 
+## RFC Process
+
+Before we invest any significant work into extending the testdriver.js API, we should check in with other stakeholders of the Web Platform Test community on the proposed changes by writing an [RFC](https://github.com/web-platform-tests/rfcs) ("request for comments"). This is especially useful for changes that may affect test authors or downstream users of web-platform-tests.
+
+The [process is given in more detail in the RFC repo](https://github.com/web-platform-tests/rfcs#the-rfc-process), but to start let's send in a PR to the RFCs repo by adding a file named `rfcs/testdriver_set_window_rect.md`:
+
+```markdown
```

Comment:
Oh it does not like this:

`/github/workspace/docs/writing-tests/testdriver-extension-tutorial.md:18:Pygments lexer name 'markdown' is not known`

Reply:
> Oh it does not like this:
> 
> `/github/workspace/docs/writing-tests/testdriver-extension-tutorial.md:18:Pygments lexer name 'markdown' is not known`

Per https://pygments.org/docs/lexers/#pygments.lexers.markup.MarkdownLexer maybe rather than `markdown`, it’s supposed to just be `md`?

Reply:
Oh thanks. I'll give that a try. If it's still grumpy, I'll just remove it. :)